tree/treeplayer/inc/ROOT/TDFOperations.hxx

```diff
@@ -33,15 +34,32 @@ using namespace Internal::TDFTraitsUtils;
 using Count_t = unsigned long;
 using Hist_t = ::TH1F;
 
+template<typename F>
+class ForeachSlotOperation {
```

Comment:
Since the fuill name is ROOT::Internal::Operations::ForeachSlotOperation, would it make sense to drop the 2nd repeat.  i.e.  ROOT::Internal::Operations::ForeachSlot and then in usage just do Operations::ForeachSlot ?